content/references/rippled-api/api-conventions/basic-data-types.md

```diff
@@ -0,0 +1,128 @@
+# Basic Data Types
+
+Different types of objects are uniquely identified in different ways:
+
+[Accounts](concept-accounts.html) are identified by their [Address][], for example `"r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59"`. Addresses always start with "r". Many `rippled` methods also accept a hexadecimal representation.
+
+[Transactions](reference-transaction-format.html) are identified by a [Hash][] of the transaction's binary format. You can also identify a transaction by its sending account and [Sequence Number][].
+
+Each closed [Ledger](reference-ledger-format.html) has a [Ledger Index][] and a [Hash][] value. When [Specifying a Ledger Instance](#specifying-ledgers) you can use either one.
+
+## Addresses
+[Address]: #addresses
+
+{% include '_snippets/data_types/address.md' %}
+
+
+## Hashes
+[Hash]: #hashes
+
+{% include '_snippets/data_types/hash.md' %}
+
+
+## Account Sequence
+[Sequence Number]: #account-sequence
+
+{% include '_snippets/data_types/account_sequence.md' %}
+
+
+## Ledger Index
+[Ledger Index]: #ledger-index
+
+{% include '_snippets/data_types/ledger_index.md' %}
+
+
+### Specifying Ledgers
+
+Many API methods require you to specify an instance of the ledger, with the data retrieved being considered up-to-date as of that particular version of the shared ledger. The commands that accept a ledger version all work the same way. There are three ways you can specify which ledger you want to use:
+
+1. Specify a ledger by its [Ledger Index][] in the `ledger_index` parameter. Each closed ledger has an identifying sequence number that is 1 higher than the previously-validated ledger. (The Genesis Ledger has sequence number 0)
+2. Specify a ledger by its [Hash][] value in the `ledger_hash` parameter.
+3. Specify a ledger by one of the following shortcuts, in the `ledger_index` parameter:
+    * `validated` for the most recent ledger that has been validated by the whole network
+    * `closed` for the most recent ledger that has been closed for modifications and proposed for validation
+    * `current` for the server's current working version of the ledger.
+
+There is also a deprecated `ledger` parameter which accepts any of the above three formats. *Do not* use this parameter; it may be removed without further notice.
+
+If you do not specify a ledger, the `current` (in-progress) ledger is chosen by default. If you provide more than one field specifying ledgers, the deprecated `ledger` field is used first if it exists, falling back to `ledger_hash`. The `ledger_index` field is ignored unless neither of the other two are present.
+
+**Note:** Do not rely on this default behavior for specifying a ledger; it is subject to change. Always specify a ledger version in the request if you can.
+
+
+## Currencies
```

Comment:
Can we find a way to split this off and merge it with the "Currency Formatting" page? I believe the two areas have a lot of redundant content.

Reply:
Maybe this is too much work and we should just add a task to the backlog to combine them.

Reply:
Created https://ripplelabs.atlassian.net/browse/DOC-1549 to track that